### PR TITLE
Add sphinx documentation build and inital API documentation.

### PR DESCRIPTION
**Pull request by hmacdope**
Partially addresses #32.

Adds in a documentation build using sphinx and a series of sphinx pages both for an introduction and start of an API reference.

Notes:
- Superseeds documentation addition on #46 
- Feedback on sphinx structure welcome. 
- We need to add a doc build and deploy to CI. ~~I will raise an issue.~~ #61 
- Still need docstrings on a bunch of things but this should be enough to get us started

### CODE REVIEW

File: fah_alchemy/settings.py
Comment (dotsdl) [inline fah_alchemy/settings.py:2]: I think this should be `fah-alchemy.settings`?

**Review by dotsdl — APPROVED** [review]
Looks great @hmacdope! Just one thing I spotted; after addressing feel free to merge!

File: fah_alchemy/settings.py
Comment (hmacdope) [reply fah_alchemy/settings.py]: Ah cheers for picking that up.

**Comment by hmacdope** [comment]
@dotsdl I will need merge permissions, which I thought I had but mustn't after all.

**Comment by dotsdl** [comment]
Hmmm, I've upped your permission level. Can you try it now?

**Comment by dotsdl** [comment]
:fireworks:

**Pull request merged**
